drivers/regulator/sy8106a.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <error.h>
+/* #include <i2c.h> */
+#include <mmio.h>
+#include <regulator/sy8106a.h>
+
+#define SY8106A_VOUT1_SEL_REG  0x01
+#define SY8106A_VOUT_COM_REG   0x02
+#define SY8106A_SYS_STATUS_REG 0x06
+
+static int
+sy8106a_disable(struct device *dev)
+{
+	uint8_t *reg;
+	int status;
+
+	status = i2c_read_reg(dev, SY8106A_VOUT_COM_REG, reg);
+	if (status != 0)
+		return -1;
```

Comment:
We have defined error codes in `error.h`. Please use them (you're including it!)

This applies to the functions below as well.

Reply:
done